docs/api/tools.md

```diff
@@ -17,7 +17,7 @@ Any transformation of the data matrix that is not *preprocessing*. In contrast t
    :nosignatures:
    :toctree: ../generated/
 
-   tl.pca
+   pp.pca
```

Comment:
I took the liberty to make this more Zen.

It’s the same function, it should be only in one place. (It’s still importable from the other, we can add a deprecation if we want, or just leave it.)